compiler/modules/front.syntax/tests/testParserOnTestSuite.bal

```diff
@@ -44,3 +63,26 @@ function partToLines(ModulePart part) returns string[] {
 function scanAndParseModulePart(string[] lines, FilePath path, int partIndex) returns ModulePart|err:Syntax {
     return parseModulePart(check scanModulePart(createSourceFile(lines, path), partIndex));
 }
+
+function testIsWhitespace(SourceFile file, Position startPos, Position endPos) returns boolean {
+    var [startLineIndex, startFragIndex] = file.fragIndex(startPos);
```

Comment:
I was suggesting to use the function that returns a [ScannedLine, int], but I now see that doesn't work because you need the index to get the lines between the startPos and endPos.

Reply:
Let me merge it and see if I can improve it.